cpp/src/rolling/rolling_detail.cuh

```diff
@@ -532,6 +528,31 @@ struct rolling_window_launcher {
                                                              following_window_begin,
                                                              min_periods);
     } else {
+      // cudf::jit::launcher(input.size(),
+      //                     cudf::jit::get_data_ptr(input),
+      //                     input.null_mask(),
+      //                     cudf::jit::get_data_ptr(output_view),
+      //                     output_view.null_mask(),
+      //                     device_valid_count.data(),
+      //                     preceding_window,
+      //                     following_window,
+      //                     min_periods);
+      // .set_kernel_inst("gpu_rolling_new",                        // name of the kernel we are
+      // launching
+      //                  {cudf::jit::get_type_name(input.type()),  // list of template arguments
+      //                   cudf::jit::get_type_name(output->type()),
+      //                   udf_agg->_operator_name,
+      //                   preceding_window_str.c_str(),
+      //                   following_window_str.c_str()})
+      //   .launch(input.size(),
+      //           cudf::jit::get_data_ptr(input),
+      //           input.null_mask(),
+      //           cudf::jit::get_data_ptr(output_view),
+      //           output_view.null_mask(),
+      //           device_valid_count.data(),
+      //           preceding_window,
+      //           following_window,
+      //           min_periods);
```

Comment:
What is this? Is it needed?

Reply:
nope. deleting. not even sure how it got there.